### PR TITLE
Installation steps for ec2 instance

### PR DESCRIPTION
**Pull request by RaviKhandavilli**
Updated Readme with instructions to set up minikube in ec2 instance and steps to enable GPU on minikube

### CODE REVIEW

File: docs/03-fiab.md [hunk 1]
```diff
@@ -109,35 +109,100 @@ sudo groupadd docker
...
+```bash
+wget https://storage.googleapis.com/golang/getgo/installer_linux 
+chmod +x ./installer_linux 
+./installer_linux 
+source ~/.bash_profile 
+ ```
+
+2. Download cri-dockerd source code.
+``` bash
+sudo -i
```
Comment (myungjin) [inline docs/03-fiab.md:133]: This should be the very first step.

**Review by myungjin — COMMENTED** [review]
one comment left. Can you please address it?

File: docs/03-fiab.md [hunk 2]
```diff
@@ -109,35 +109,100 @@ sudo groupadd docker
 sudo usermod -aG docker $USER && newgrp docker 
 ```
 
+### AWS EC2 instance with Amazon Linux 2 image and GPU
+This section is specifically for AWS EC2 instance with GPU. For general linux machines without GPU (regardless of VM or baremetal machine),
+the guideline for [linux](#linux) is recommended.
+For Amazon linux 2 image (amzn2), the following tools are necessary: `minikube`, `kubectl`, `helm`, `cri-dockerd`, `crictl` , `docker` and `jq`.
+The image was tested under an ec2 instance with GPU (e.g., p2 instances). The following procedures show how to enable GPU support in minikube:
 
-## Starting minikube
```
Comment (myungjin) [inline docs/03-fiab.md:113]: Why are all of these removed? These are still needed to enable fiab. We shouldn't delete them.

**Review by myungjin — COMMENTED** [review]
one more feedback left.

File: docs/03-fiab.md (hunk 2 above)
Comment (RaviKhandavilli) [reply docs/03-fiab.md:113]: corrected

**Review by myungjin — APPROVED** [review]
lgtm

**Pull request merged**